docs/development/CODE_REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,121 @@
+
+If the commits be (re)organized (i.e. reordered and/or amended) such that there is a commit at which the tests are passing prior to the conclusion of the main change, that's a signal that there's likely a logic split which can be made at that point in such a (re)arrangement.
+
+Use the following guidelines to evaluate whether a Pull Request should be approved:
+
+- *Every* Pull Request should have tests or justification otherwise; esp. bug fixes.
+- *Every* Pull Request should have at least 1 approval from a team member internal or external to the project. Exceptions made by repository maintainer(s), as necessary, on a case-by-case basis.
+
+During review, submit feedback using line comments (A); prefer "Add\[ing a\] single comment" over "Start[ing] a review" (B).
+
+*(Once a review has been started, the option to add single comments is removed. Preferring single comments allows feedback to happen even in the event of an interrupted review.)*
+
+### A
+
+![github_line_comment.png](./assets/github_line_comment.png)
+
+### B
+
+![line_comment_dialog_start.png](./assets/line_comment_dialog_start.png)
+
+**Referencing Issues Across Repositories:**
+When referencing issues from one repository, in another's Issues and Pull Requests, GitHub supports automatic links in markdown using the following format: `<org name>/<repo name>#<issue or PR number>`.`
+
+### Finishing a Review
+
+Write a summary of your observations and consider including positive remarks in addition to any constructive criticism.
+
+If you observe a deviation from these practices or another reason that this change should not be merged, select "request changes" and include a summary of the observation(s), as well as any practice(s) you find them to be in conflict with, in the review body (C).
+
+If you don't feel comfortable giving approval or requesting changes but want to share a summary or observations of larger patterns in the codebase or the company, select "Comment" and submit your review (C).
+
+Confirm that all items in the "required checklist" are checked or not applicable.
+
+If you believe the Pull Request looks good to merge, select "Approve" and submit your review (C).
+
+### C
+
+![submit_review.png](./assets/submit_review.png)
+
+### Merging
+
+1. Utilize the `Squash & Merge` feature (maintain a clean history)
+2. Copy the `Github PR Description` into the commit message (add sufficient detail)
+
+**Authors are core members or regular external contributors**
+
+- Core member needs to approve PR
+- Author should merge in the PR themselves (following instructions above)
+
+**Authors are non-regular external contributors:**
+
+- Core member needs to approve PR
+- Core member can merge PR on behalf of contributor  (following instructions above)
+
+**Pull Request Authors:**
+Specify whether or not the Pull Request can be merged by the last approving reviewer by use of (or lack thereof) the "Reviewer can merge" label. If no such label exists for a given project, create it.
```

Comment:
Is this `Reviewer can merge` label a new idea? Can you add it if this is the pattern we'll use?

Reply:
New to Pocket, yes. Unless there are any principled objections, I would like for us to give it a try.

Reply:
No objections, let's do it.